projects/knora-ui/package.json

```diff
@@ -1,6 +1,11 @@
 {
-  "name": "knora-ui",
-  "version": "1.0.0-alpha",
+  "name": "@knora/ui",
+  "version": "1.0.0-rc1",
```

Comment:
I think it should be 1.0.0-rc.1, see https://stackoverflow.com/questions/50846170/how-to-generate-npm-release-candidate-version

Reply:
https://docs.npmjs.com/misc/semver#prerelease-tags -> they also use this notation

Reply:
https://semver.org/#spec-item-11

Reply:
to be precise, it should be `1.0.0-rc.0` 😄
I followed your links and ran the command `npm version prerelease --preid=rc` (from one stackoverflow response) in `projects/knora-ui` and I was getting the version `1.0.0-rc.0`